app/views/taxons/sections/_guidance_and_regulation.html.erb

```diff
@@ -1,5 +1,6 @@
 <%= render 'components/highlight-boxes',
-  items: section[:documents].shift(section[:promoted_content_count])
+  items: section[:documents].shift(section[:promoted_content_count]),
+  data_track_category: "guidanceAndRegulationHighlightBoxClicked"
```

Comment:
This value feels like something that should be set in the supergroup presenter. It could also probably be created programmatically.